media/css/firefox/whatsnew/whatsnew-127-eu.scss

```diff
@@ -0,0 +1,108 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$brand-theme: "firefox";
+
+@import "includes/base";
+@import "~@mozilla-protocol/core/protocol/css/includes/lib";
+@import "~@mozilla-protocol/core/protocol/css/components/notification-bar";
+@import "~@mozilla-protocol/core/protocol/css/components/video";
```

Comment:
**Suggestion:** if you replace this line with `@import '../../protocol/components/video';`, then you might not need the `.video-play` styles on line 74? 
```suggestion
@import '../../protocol/components/video';
```